parsing/location.ml

```diff
@@ -423,10 +438,8 @@ let () =
       | Sys_error msg ->
           Some (errorf ~loc:(in_file !input_name)
                 "I/O error: %s" msg)
-      | Warnings.Errors n ->
-          Some
-            (errorf ~loc:(in_file !input_name)
-             "Some fatal warnings were triggered (%d occurrences)" n)
+
+      | Warnings.Errors _ -> Some non_displayed_error
```

Comment:
This seems a bit of a hack. What about capturing Warning.Errors directly in Location.report_exception (and just do nothing)?